src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -48,8 +50,8 @@
   []
   (rf/dispatch [:syncing/preflight-outbound-check #(reset! preflight-check-passed? %)]))
 
-(defn- f-header
-  [{:keys [active-tab read-qr-once? title title-opacity subtitle-opacity reset-animations-fn animated?]}]
```

Comment:
`read-qr-once?` was an atom to control the callbacks dispatched by the camera-kit component, it dispatched the `on-read` function many times before we navigated.

Now it's not required